src/Uno.PackageDiff/Program.cs

```diff
@@ -148,7 +148,11 @@ private static async Task<(string path, NuspecReader nuspecReader)> GetPackage(s
 
 					if (latestStable != null)
 					{
-						var url = $"https://api.nuget.org/v3-flatcontainer/{packagePath}/{latestStable.Version}/{packagePath}.{latestStable.Version}.nupkg";
+						var packageId = packagePath.ToLowerInvariant();
+						var version = latestStable.Version.ToString().ToLowerInvariant();
```

Comment:
```suggestion
						var version = latestStable.Version.ToNormalizedString().ToLowerInvariant();
```

Reply:
Thanks for the review! I'm not familiar enough with the APIs of this package. I search for a direct download link in the APIs, but most of those require a local nuget cache to work with.

Reply:
Yeah, your current approach is workable. The `ToNormalizedString()` is important though. Non-normalized versions will not be recognized by the download URL and could yield a 404.

e.g. non-normalized version `1.0.0.0-BETA`. Normalization would yield `1.0.0-BETA`. The string used in the URL would be `1.0.0-beta`.